datastream-server-restli/src/main/java/com/linkedin/datastream/server/diagnostics/ServerComponentHealthAggregator.java

```diff
@@ -92,8 +92,7 @@ public ServerComponentHealthAggregator(ZkClient zkClient, String cluster, int en
         if (!errorMessage.isEmpty()) {
           errorResponses.put(hostName, errorMessage);
         } else {
-          String message = "Received REST response from the host: " + dmsUri + " with status: " + response.getStatus();
-          LOG.info(message);
+          LOG.info("Received REST response from the host: " + dmsUri);
```

Comment:
Can you please add the response in the debug mode, in case we need to enable and verify it?